### PR TITLE
fix: preserve background color through reload

### LINKED ISSUE #14976: BrowserWindow background color is white after reload
* Output of `node_modules/.bin/electron --version`: v3.0.2
* Operating System (Platform and Version): Windows 10 Pro 17134
* Output of `node_modules/.bin/electron --version` on last known working Electron version (if applicable): v1.6.15

**Expected Behavior**
Reloading (Ctrl R) a `BrowserWindow` with `backgroundColor` set should display that background color until the page begins to load.

**Actual behavior**
Reloading (Ctrl R) a `BrowserWindow` with `backgroundColor` set displays white until the page begins to load.

**To Reproduce**
I've reproduced the issue in electron-quick-start by:
* Adding ` style="background-color: transparent;"` to `index.html`'s `body` tag (pedantic, I know)
* Adding `backgroundColor: '#ffff0000', ` to `main.js`'s `BrowserWindow` options
* Alternatively, clone my fork of electron-quick-start:
```sh
$ git clone https://github.com/adill/electron-quick-start.git -b background-color-reload
$ npm install
$ npm start
```
1. Notice the red background color on the page (transparent body is showing through to the opaque red background.)
2. Reload (Ctrl R)
3. Notice the background color is now white

### PR DESCRIPTION
**Pull request by adill**
##### Description of Change

Fixes #14976.

##### Checklist

- [x] PR description included and stakeholders cc'd
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)

##### Release Notes

Notes: fix: backgroundColor is preserved through reload

### CODE REVIEW

File: atom/renderer/renderer_client_base.cc [hunk 1]
```diff
@@ -180,22 +180,26 @@ void RendererClientBase::RenderFrameCreated(
   blink::WebSecurityPolicy::AddOriginAccessWhitelistEntry(
       GURL(kPdfViewerUIOrigin), "file", "", true);
 #endif  // BUILDFLAG(ENABLE_PDF_VIEWER)
+
+  content::RenderView* render_view = render_frame->GetRenderView();
```
Comment (deepak1556) [inline atom/renderer/renderer_client_base.cc]: This method will be called for all frames created in this process, we should make sure that bg color applies only when the top level frame is created, can you add a condition to ensure that. With `RenderViewCreated` it was implicit.

Comment (adill) [reply atom/renderer/renderer_client_base.cc]: 👍 I've guarded it to only apply when `render_frame->IsMainFrame()`

**Comment by codebytere** [comment]
@adill these test failures look to be related:
screen shot 2018-10-13 at 7 58 13 am

**Comment by codebytere** [comment]
@adill can you please open up a backport of this PR targeting `3-0-x`?

**Comment by adill** [comment]
@codebytere https://github.com/electron/electron/pull/15175

**Pull request merged**